1-exercises/A-array-find/exercise.js

```diff
@@ -17,6 +17,9 @@ let names = [
   "Ahmed",
 ];
 
+const findLongNameThatStartsWithA = (names) =>
```

Comment:
Great! What is happened if you don't use arrow function before your find method?